ee-feature-pack/galleon-shared/src/main/resources/feature_groups/remoting.xml

```diff
@@ -2,9 +2,7 @@
 <feature-group-spec name="remoting" xmlns="urn:jboss:galleon:feature-group:1.0">
     <!-- TODO Temporary fork to override use of security-realms. -->
```

Comment:
This can be updated, as it's now a fork to add the http-connector.

Reply:
Updated.